src/Smithers.js

```diff
@@ -22,4 +22,34 @@ export default class Smithers {
         const requestUrl = createPath(paths.lastBuild, { name });
         return this.caller.get(requestUrl, config);
     }
+
+    lastSuccessfulBuild(name, config) {
+        const requestUrl = createPath(paths.lastSuccessfulBuild, { name });
+        return this.caller.get(requestUrl, config);
+    }
+
+    lastStableBuild(name, config) {
+        const requestUrl = createPath(paths.lastStableBuild, { name });
+        return this.caller.get(requestUrl, config);
+    }
+
+    lastUnsuccessfulBuild(name, config) {
+        const requestUrl = createPath(paths.lastUnsuccessfulBuild, { name });
+        return this.caller.get(requestUrl, config);
+    }
+
+    specificBuild(name, build, config) {
+        const requestUrl = createPath(paths.specificBuild, { name, build });
+        return this.caller.get(requestUrl, config);
+    }
+
+    overallLoad(config) {
+        const requestUrl = createPath(paths.overallLoad);
```

Comment:
No need to put `paths.overallLoad` through the createPath method as no variable injection required.